src/snyk/base/modules/snykLib.ts

```diff
@@ -136,10 +136,6 @@ export default class SnykLib extends BaseSnykModule implements ISnykLib {
       if (result instanceof CliError || !result) {
         return;
       }
-
```

Comment:
Do we need line 133 call and the actual getter now?*

_* couldn't add comment on the proper line, since GitHub errors._

Reply:
nope, it can be removed